lua/nvim-tree/view.lua

```diff
@@ -224,7 +224,16 @@ function M.open()
   a.nvim_command("vsp")
   local move_to = move_tbl[M.View.side]
   a.nvim_command("wincmd "..move_to)
-  a.nvim_command("vertical resize "..M.View.width)
+
+  local width
+  if (vim.g.nvim_tree_width_as_percent == true) then
+    local percentAsDecimal = M.View.width/100
+    width = math.floor( vim.o.columns * percentAsDecimal )
```

Comment:
remove spaces after opening parens and before closing parens

Reply:
Could just add some formatter to CI checking each PR for style issues. It's good to have consistent styling throughout the project.